src/main/kotlin/com/terraformation/backend/tracking/model/BiomassDetailsModel.kt

```diff
@@ -12,24 +12,28 @@ import com.terraformation.backend.db.tracking.TreeGrowthForm
 import java.math.BigDecimal
 import java.time.Instant
 
-data class BiomassAdditionalSpeciesModel(
+data class BiomassSpeciesKey(
+    val speciesId: SpeciesId? = null,
+    val scientificName: String? = null,
+)
```

Comment:
This is totally fine, but if you like, you could define this in the `EMBEDDABLES` list in `Config.kt` and then jOOQ will include it as a field in the generated classes so you don't have to construct it manually from the ID and name fields.